### PR TITLE
friendship v-1

### PR DESCRIPTION
**Pull request by atenaiis**
Team

The aim is to create a model with associations and all requested features for friendships.

cheers!!

### CODE REVIEW

**Review by rukundoeric — CHANGES_REQUESTED** [review]
# Status: ♻️ CHANGES REQUIRED ♻️
_Great job guys so far 👏 💯 👍. However, there are few issues we need to address before we move on, please check the comments below._

- ### To Highlight
  - [x] The code looks clean and professional. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - [x] [MANDATORY] There is a test case that is failing. checkout this error: 
     ```
     NameError:
       undefined local variable or method `friendships_path' for #<RSpec::ExampleGroups::Friendships::GETFriendships:0x00007f8fa5f0c260>
     ```

_Please make all the required changes before you submit another code review. If you have any problems or concerns, you can reach out to me on Slack: Rukundo Eric 🚀 . I wish you all the very best 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/), 
Slack: Rukundo Eric**

**Comment by atenaiis** [comment]
thank you for your comments.

I added the test for friendship_spec.

cheers!!!

File: spec/models/friendship_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (umairarshadbutt) [inline spec/models/friendship_spec.rb]: Please remove the `pending` statement.

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @atenaiis 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes  👇.
- [x] There is a one test case is pending at `spec/models/friendship_spec.rb`. You can write it or remove it.
**OPTIONAL**
- [x] Your linter checks (GitHub Actions) are not running. Kindly ensure that all checks are running successfully.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by atenaiis** [comment]
Thanks for your comments

I removed the file .

Cheers!! :) :+1:

**Review by BrianSammit — APPROVED** [review]
STATUS: APPROVED :green_circle: :clap: 
-------------
Hello @atenaiis you did an amazing job working on the required changes :muscle: :100: 
------------
Now you can merge this branch :100: :+1: 
HAPPY CODING!! :smile: 
Well-Done-Applause

**Pull request merged**